Makefile

```diff
@@ -22,6 +22,11 @@ MD ?= @
 # TLS related configuration
 XI_BSP_TLS ?= wolfssl
 
+#detect if the build happen on Travis
+ifdef TRAVIS_OS_NAME
```

Comment:
I saw your previous version where you used the TRAVIS_OS_NAME directly, why did you make this plus step?